docs/develop/yeoman-generator-overview.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Create Office Add-in projects with the Yeoman Generator
+description: Learn how to create Office Add-in projects with the Yeoman generator for Office Add-ins.
+ms.date: 03/11/2022
+ms.localizationpriority: high
+---
+
+# Create Office Add-in projects with the Yeoman Generator
+
+The Yeoman Generator for Office Add-ins (also called "Yo Office") is an interactive Node.js-based command line tool that creates Office Add-in development projects. We recommend that you use this tool to create add-in projects except when you want the server-side code of the add-in to be in a .NET-based language (such as C# or VB.Net) or you want the add-in hosted in Internet Information Server (IIS). In either of the latter two situations, [use Visual Studio to create the add-in](develop-add-ins-visual-studio.md).
+
+The projects that the tool creates have the following characteristics:
+
+- They have a standard [npm](https://www.npmjs.com/) configuration that includes a **package.json** file.
+- They include several helpful scripts to build the project, start the server, sideload the add-in in Office, and other tasks.
+- They use [webpack](https://webpack.js.org/) as a bundler and basic task runner.
+- In development mode, they are hosted on localhost by webpack's Node.js-based webpack-dev-server, a development-oriented version of the [express](http://expressjs.com/) server that supports hot-reloading and recompile-on-change.
+- By default, all dependencies are installed by the tool, but you can postpone the installation with a command line argument.
+- They include a complete add-in manifest.
+- They have a "Hello World"-level add-in that can be run as soon as the tool has finished.
+- They include a polyfill and a transpiler that is configured to transpile TypeScript, and recent versions of JavaScript, to ES5 JavaScript. These features ensure that the add-in is supported in all browser runtimes that Office Add-ins might run in, including Internet Explorer.
+
+> [!TIP]
+> If you want to deviate from these choices significantly, such as using a different task runner or a different server, we recommend that when you run the tool you take the option described in [Manifest-only option](#manifest-only-option).
+
+## Install the generator
+
+[!include[Set up requirements](../includes/set-up-dev-environment-beforehand.md)]
+[!include[Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
+
+## Use the tool
+
+Start the tool with the following command in a system prompt (not a bash window):
+
+```command&nbsp;line
+yo office 
+```
+
+A lot needs to load, so it may take 20 seconds before the tool starts. The tool asks you a series of questions. For some, you just type an answer to the prompt. For others, you're given a list of possible answers. If given a list, select one using the arrow keys, then select the Enter key.
```

Comment:
You specifically point out that the user is using the keyboard by referring to the arrow keys. So, since you're referring to using keyboard only, we can use "press". Otherwise, just say "If given a list, select one and choose Enter." So you're not referring to the keyboard. You're right, there are multiple ways to interact with the UI, so we should remove mention of both arrow keys and pressing enter.